docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,808 @@
+# Working with the CHIP Tool
+
+The CHIP Tool (`chip-tool`) is a Matter controller implementation that allows to
+commission a Matter device into the network and to communicate with it using
+Matter messages, which may encode Data Model actions, such as cluster commands.
+The tool also provides other utilities specific to Matter, such as parsing of
+the setup payload or performing discovery actions.
+
+<hr>
+
+-   [Source files](#source)
+-   [Building and running the CHIP Tool](#building)
+-   [Using the CHIP Tool for Matter device testing](#using)
+-   [Supported commands and options](#commands)
+
+<hr>
+
+<a name="source"></a>
+
+## Source files
+
+You can find source files of the CHIP Tool in the `examples/chip-tool`
+directory.
+
+> **Note:** The CHIP Tool currently only supports commissioning and remembering
+> one device at a time. The configuration state is cached in the
+> `/tmp/chip_tool_config.ini` file. Deleting this and other `.ini` files in the
+> `/tmp` directory can sometimes resolve issues related to stale configuration.
+
+<hr>
+
+<a name="building"></a>
+
+## Building and running the CHIP Tool
+
+Before you can use the CHIP Tool, you must compile it from source on Linux
+(amd64/aarch64) or macOS. If you want to build on Raspberry Pi, it must run
```

Comment:
I will slightly change the Rasp's OS requirement. The OS must be Ubuntu but the recommended version is 20.04